devicon.json

```diff
@@ -2214,6 +2214,37 @@
         "color": "#DF234F",
         "aliases": []
     },
+    {
+        "name": "nextjs",
+        "tags": [
+            "framework"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "line",
+                "original-wordmark"
+            ],
+            "font": [
+                "original",
+                "plain",
```

Comment:
There are no `plain` or `plain-wordmark` versions. Please add a `plain` (if possible `plain-wordmark`, too) version.

Reply:
I thought I have done that already with the aliases, but I am not quite sure if that is how it works. Can you maybe help here @amacado?

Reply:
@maltejur is correct. While physically, there aren't any `line` or `line-wordmark` svg files, our build script will create a CSS class for `line` and `line-wordmark` that will point to the `original` and `original-wordmark` respectively.

It's nice that you remember it! We are not enforcing aliases anymore but it's always useful to have it.